string m_name
float32 m_trust
float32 m_trustLast
string[] m_detectedBySensorId
# maybe int type?
